### DISCUSSION TITLE
New feature: engine problems

### DISCUSSION THREAD

**Discussion by WilliamDelamare** [Estia-advanced-programming/pandora-public#86] [Ideas for features]
Create an issue to find flights with engine problems 

* set some records engines to 0

**Comment by WilliamDelamare** [comment]
(the goal is to also make engines computations (avg, max, etc) more tricky)

**Comment by dhmmasson** [comment]
Fun, I was thinking of adding some difference in the engine power in my tests also.
We could add the stddev ?
